src/admin-views/admin-welcome-message.php

```diff
@@ -2,84 +2,213 @@
+		<div class="tribe-events-admin-card tribe-events-admin-card--3up tribe-events-admin-card--middle">
+			<img
+				class="tribe-events-admin-card__image"
+				src="<?php echo esc_url( tribe_resource_url( 'images/welcome/knowledgebase.jpg', false, null, $main ) ); ?>"
+				alt="<?php esc_attr_e( 'illustration of a thought lightbulb coming from a book', 'event-tickets' ); ?>"
+			/>
+			<div class="tribe-events-admin-card__title"><?php esc_html_e( 'Knowledgebase', 'event-tickets' ); ?></div>
+			<div class="tribe-events-admin-card__description"><?php esc_html_e( 'Ready to dig deeper? Our Knowledgebase can help you get the most out of The Events Calendar suite.', 'event-tickets' ); ?></div>
+			<a class="tribe-events-admin-card__link" href="https://m.tri.be/1ane" target="_blank" rel="noopener noreferrer"><?php esc_html_e( 'Dig deeper', 'event-tickets' ); ?></a>
+		</div>
+		<div class="tribe-events-admin-card tribe-events-admin-card--3up tribe-events-admin-card--last">
+			<img
+				class="tribe-events-admin-card__image"
+				src="<?php echo esc_url( tribe_resource_url( 'images/welcome/translations.jpg', false, null, $main ) ); ?>"
+				alt="<?php esc_attr_e( 'illustration of characters being translated', 'event-tickets' ); ?>"
+			/>
+			<div class="tribe-events-admin-card__title"><?php esc_html_e( 'Translations', 'event-tickets' ); ?></div>
+			<div class="tribe-events-admin-card__description"><?php esc_html_e( 'Need a language other than English? We\'ve got you covered here.', 'event-tickets' ); ?></div>
+			<a class="tribe-events-admin-card__link" href="https://m.tri.be/language" target="_blank" rel="noopener noreferrer"><?php esc_html_e( 'Learn more', 'event-tickets' ); ?></a>
+		</div>
 
-	<div class="tribe-half-column">
-		<h4 data-tribe-icon="dashicons-megaphone"><?php esc_html_e( "Don't Miss Out", 'event-tickets' ); ?></h4>
-		<p><?php esc_html_e( 'Stay in touch with Event Tickets and our entire family of events management tools. We share news, occasional discounts, and hilarious gifs.', 'event-tickets' ); ?></p>
+		<div class="tribe-events-admin-card tribe-events-admin-card--1up">
+			<img
+				class="tribe-events-admin-card__image"
+				src="<?php echo esc_url( tribe_resource_url( 'images/welcome/virtual-events.jpg', false, null, $main ) ); ?>"
+				alt="<?php esc_attr_e( 'illustration of a phone screen with a person\'s face', 'event-tickets' ); ?>"
+			/>
+			<div class="tribe-events-admin-card__title"><?php esc_html_e( 'Virtual Event Resources', 'event-tickets' ); ?></div>
+			<div class="tribe-events-admin-card__description"><?php esc_html_e( 'Tips and tools to help with planning online events, webinars, and more on WordPress and beyond.', 'event-tickets' ); ?></div>
+			<a class="tribe-events-admin-card__link" href="https://m.tri.be/1ani" target="_blank" rel="noopener noreferrer"><?php esc_html_e( 'Get started with online events', 'event-tickets' ); ?></a>
+		</div>
+
+		<?php if ( $has_plus && $has_tec ) : // ET+ with TEC ?>
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 15a67a78b3179e6c15f2c6eb35f626a0</sub>